contributing/code/standards.rst

```diff
@@ -172,6 +172,10 @@ Documentation
 
 * Add PHPDoc blocks for all classes, methods, and functions;
 
+* Group annotations together so that annotations of the same type immediately
```

Comment:
I think this proposal makes sense ... but in some cases it can be difficult to decide the annotation type. `@ORM` and `@Assert` obviously define their own groups, but what about `@Cache`, `@Method`, `@Route`, `@Security` Are these four annotations of the same type?